src/main/index.ts

```diff
@@ -91,10 +91,13 @@ function createMainWindow() {
 
   window.once("ready-to-show", () => {
     didFinishLoad = true;
-    window.show();
-    window.focus();
+    //window.show();
+    //window.focus();
   });
 
+  window.show();
+  window.focus();
```

Comment:
Why move these out of the ready-to-show event? Isn't that what the event is meant for?

Reply:
The window itself can show and things pop in as they're ready. I kept the `didFinishLoad` param for the place where something is waiting for the ready event; this functionally means the window shows quickly -> app loads and renders in.

It's a small distinction but feels much quicker.

Reply:
To add: I anticipate on Windows/Linux, where the _window_ represents the _app_, this isn't felt too much - sometimes apps take time to open. On macOS, reopening a window should feel quick.

Reply:
(There's no odd paint flash as `backgroundColor` is passed to the window constructor, so that helps alleviate the common case where you would need to show the window after this event)

Reply:
Paint flash happens on windows so we should keep it in ready-to-show if not on macOS.

Reply:
Kk, done.